### ISSUE TITLE
Root namespace in the child nodes

### ISSUE THREAD

**Issue by suleman-uzair** [lutaml/lutaml-model#96]
The namespace of the root element is passed onto child elements using OxAdapter:
```xml
<math xmlns="http://www.w3.org/1998/Math/MathML">
  <mstyle xmlns="http://www.w3.org/1998/Math/MathML">
    <mi xmlns="http://www.w3.org/1998/Math/MathML">a</mi>
  </mstyle>
</math>
```
Whereas, NokogiriAdapter does not:
```xml
<math xmlns="http://www.w3.org/1998/Math/MathML">
  <mstyle>
    <mi>a</mi>
  </mstyle>
</math>
```

**Comment by ronaldtse** [comment]
I think the NokogiriAdapter behavior is correct here, where the default namespace is assumed by the child nodes. What does the XML Schema spec say?

**Comment by ronaldtse** [comment]
@suleman-uzair is this addressed?

**Comment by suleman-uzair** [comment]
> is this addressed?

@ronaldtse, Yes, It has been fixed in version `0.3.24`, see the attached screenshot for the processed example.

![Screenshot 2024-11-14 at 1 29 59 PM](https://github.com/user-attachments/assets/0def99bd-32c9-4136-813a-f6668fd09367)
